src/components/charts/PositionsComparisonChart.js

```diff
@@ -0,0 +1,60 @@
+import React, { useMemo } from 'react';
+import { LineChart, Line, XAxis, YAxis, CartesianGrid, Tooltip, Legend, ResponsiveContainer } from 'recharts';
+import { lightenColor } from '../../utils/lightenColor';
+
+const CustomizedXAxisTick = ({ x, y, payload }) => {
```

Comment:
seems like we can pull this into a sharable file as well right, since its duplicated in each.

Reply:
yes, this was already made into a sharable file no? for both the race and quali positions